iphone/Maps/UI/Storyboard/Main.storyboard

```diff
@@ -3,7 +3,7 @@
     <device id="retina6_72" orientation="landscape" appearance="light"/>
     <dependencies>
         <deployment identifier="iOS"/>
-        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="22684"/>
+        <plugIn identifier="com.apple.InterfaceBuilder.IBCocoaTouchPlugin" version="22685"/>
```

Comment:
You can safely remove all this autogenerated boilerplate.

Reply:
I can revert it, but next person editing this file on modern Xcode (15.4 in my case) will again auto-update this version I guess? Maybe just let it be updated?

Reply:
Ok! Let it be. But not everyone uses the latest xcode version. 
As for me, it's better to avoid committing the code generated by just opening the IB screen fact without any payload. 
I think that in the future (when we up the target at least to ios13) we can start replacing all these storyboards with SUI step-by-step because its hard to support xibs sometimes and almost impossible to review.